src/test/resources/overseas_entity_v_3_2.json

```diff
@@ -783,7 +783,8 @@
             "locality" : "string"
           },
           "date_became_interested_person" : "2022-06-10",
-          "ceased_date" : "2024-02-29"
+          "ceased_date" : "2024-02-29",
+          "is_individual_still_involved_in_trust" : true
```

Comment:
Minor, but I'd make the value here `false`, just so the data-set makes sense (you wouldn't have a ceased date if individual was still involved with the trust).

Reply:
changed